docs/source/getting-started-cluster-mode.md

```diff
@@ -17,17 +17,17 @@ SPARK_HOME:/usr/hdp/current/spark2-client                            #For HDP di
 EG_YARN_ENDPOINT=http://${YARN_RESOURCE_MANAGER_FQDN}:8088/ws/v1/cluster #Common to YARN deployment
 ``` 
 
-**Configuring Kernels for YARN Cluster mode**
+### Configuring Kernels for YARN Cluster mode
 
 For each supported Jupyter Kernel, we have provided sample kernel configurations and launchers as part of the release
-[e.g. jupyter_enterprise_gateway_kernelspecs-0.6.0.tar.gz](https://github.com/jupyter-incubator/enterprise_gateway/releases/download/v0.6.0/jupyter_enterprise_gateway_kernelspecs-0.6.0.tar.gz).
+[e.g. jupyter_enterprise_gateway_kernelspecs-0.9.1.tar.gz](https://github.com/jupyter-incubator/enterprise_gateway/releases/download/v0.9.1/jupyter_enterprise_gateway_kernelspecs-0.9.1.tar.gz).
```

Comment:
Is there a way to have the version be something like a macro? In this case, it should actually be the current master version and not a specific release.

Reply:
@lresende - no idea.  We can look into this.  However, these artifacts are manually uploaded to the releases directory when tied to a given release - so I'm not sure how someone would pull "today's build".